java/src/jmri/jmrix/ecos/utilities/AddRosterEntryToEcos.java

```diff
@@ -29,12 +31,15 @@
   */
 public class AddRosterEntryToEcos extends AbstractAction {
 
+    private EcosLocoAddressManager objEcosLocoManager;
+
     /**
      * @param s Name of this action, e.g. in menus
      */
     public AddRosterEntryToEcos(String s, EcosSystemConnectionMemo memo) {
         super(s);
         adaptermemo = memo;
+        objEcosLocoManager = adaptermemo.getLocoAddressManager();
```

Comment:
I think this line is causing the FindBugs errors by requiring that ```memo``` be non-null. This constructor is called by EcosMenu (line 48), which is called by SystemsMenu (line 92), and since this line would throw an NPE if memo is null, that would propagate up to SystemsMenu which passes ```null``` as the memo.